tree/bst/height.py

```diff
@@ -0,0 +1,60 @@
+"""
+Write a function height returns the height of a tree. The height is defined to
+be the number of levels. The empty tree has height 0, a tree of one node has
+height 1, a root node with one or two leaves as children has height 2, and so on
+For example: height of tree is 4
+
+                    9
+                 /      \
+               6         12
+              / \       /   \
+            3     8   10      15
+                 /              \
+                7                18
+
+    height = 4
+
+"""
+import unittest
+from bst import Node
+from bst import bst
+
+def height(root):
+    if root is None:
+        return 0
+    else:
+        return max(1 + height(root.left), 1 + height(root.right))
```

Comment:
Please change this to the following:
return 1 + max(height(root.left), height(root.right))

Reply:
Try doing
``` python

# If you mean continuation, use the following.
return A + B + \
  C + D

# The below returns 'abc'
return reduce(operator.concat, ['a', 'b', 'c'])

# There are other ways to do the continuation : 'abc'
return ''.join(['a', 'b', 'c'])
```

Reply:
@danghai Or you can simply do
```python
return (A + B
            + C + D)
```